revenue_app/static/revenue_app/js/generate_pdf.js

```diff
@@ -0,0 +1,66 @@
+function generatePDF() {
+
+  //Get svg markup as string
+  var titleArs = document.getElementById('title-ars');
+  var tableArs = document.getElementById('dynamic-table-ars');
+  var svgChartArs = document.getElementById('chart-top-ars').innerHTML;
+  var svgLegendArs = document.getElementById('legend-top-ars').innerHTML;
+
+  svgChartArs = svgChartArs.replace(/\r?\n|\r/g, '').trim();
+  svglegend = svgLegendArs.replace(/\r?\n|\r/g, '').trim();
+
+  var canvasChartArs = document.createElement('canvas');
+  var canvasLegendArs = document.createElement('canvas');
```

Comment:
maybe we can work on decoupling this, to avoid additional custom titles or references to currencies